libs/market-list/src/lib/components/asset-details-dialog/asset-details-dialog.tsx

```diff
@@ -0,0 +1,194 @@
+import { gql, useQuery } from '@apollo/client';
+import { formatNumber, t, toBigNum } from '@vegaprotocol/react-helpers';
+import type { Asset } from '@vegaprotocol/react-helpers';
+import {
+  Button,
+  Dialog,
+  Icon,
+  Intent,
+  Splash,
+  Tooltip,
+} from '@vegaprotocol/ui-toolkit';
+import type {
+  AssetsConnection,
+  AssetsConnection_assetsConnection_edges_node_source_ERC20,
+} from './__generated__/AssetsConnection';
+
+export type AssetDetailsDialogState = {
+  isAssetDetailsDialogOpen: boolean;
+  assetDetailsDialogSymbol: string | Asset;
+};
+
+type AssetDetails = {
+  key: string;
+  label: string;
+  value: string;
+  tooltip: string;
+}[];
+
+export const DEFAULT_ASSET_DETAILS_STATE: AssetDetailsDialogState = {
+  isAssetDetailsDialogOpen: false,
+  assetDetailsDialogSymbol: '',
+};
+
+export const ASSETS_CONNECTION_QUERY = gql`
+  query AssetsConnection {
```

Comment:
suggestion (non blocking): I would consider passing in the asset id as a prop and querying for a single asset using it

```
query Asset {
  asset(assetid: ID!) {
    ...
  }
}
```

Reply:
follow up in #1049